frontend/src/methods/key.ts

```diff
@@ -262,6 +262,10 @@ const registerInterceptors = () => {
         } else if (url.indexOf("/image/") == 0) {
           config.headers.set(TimestampHeaderKey, ts);
 
+          if (config.method == "HEAD") {
+            config.headers.set(`Cache-Control`, "no-store");
+          }
```

Comment:
This works, but I'd rather suggest to add this header here:

https://github.com/siderolabs/omni/blob/main/frontend/src/views/omni/Modals/DownloadInstallationMedia.vue#L323

Reply:
Do you mean to add a third option argument with headers and pass it from this place?

Reply:
Remove this `if`.
And change the line I posted to:
```
    await doRequest(url, { signal: controller.signal, method: "HEAD", headers: { "Cache-Control": "no-store" } });
```

will do the trick

Reply:
So yeah, I guess that what you mean :)

Reply:
Done! Didn't know that I can do that, sorry, I'm not a TS developer :D

Reply:
None of us is a TS developer here :D

Looks like my suggestion wasn't correct. But I'll fix that.